packages/generator-clay/app/templates/src/__tests__/__snapshots__/_Boilerplate.js.snap

```diff
@@ -0,0 +1,4 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I wouldn't add snapshots to the generated component. You will need to update it once you write your code so...

Reply:
All your concern look very valid to me, but I reckon it'd be rather handy for some developers to at least show the structure of how the snapshots should be put together, though I understand it must be updated and aligned to the component anyway.

Reply:
As Westley would say in "The Princess Bride": As you wish :P

Reply:
Thanks :)